docs/releases/2.2.3.txt

```diff
@@ -15,3 +15,7 @@ Bugfixes
 
 * Fixed a regression in Django 2.2.2 where auto-reloader crashes with
   ``AttributeError``, e.g. when using ``ipdb`` (:ticket:`30588`).
+
+* Fixed a bug where in multi-table inheritance, serializers output
+  :class:`~django.db.models.ManyToManyField` in both the parent and the child
+  class. (:ticket:`28725`).
```

Comment:
This doesn't qualify for a backport, so you can remove release notes.

Reply:
Ok, should I put that in docs/release/3.0.txt instead ?

Reply:
We don't need any release notes. 3.0 is a feature release and we don't mention bugfixes in such releases.

Reply:
oh ok, thank you :)